drivers/storage/rackspace/executor/rackspace_executor.go

```diff
@@ -0,0 +1,354 @@
+package executor
+
+// // StorageExecutor is the part of a storage driver that is downloaded at
+// // runtime by the libStorage client.
+// type StorageExecutor interface {
+// 	Driver
+// 	StorageExecutorFunctions
+// }
+
+// // StorageExecutorFunctions is the collection of functions that are required
+// // of a StorageExecutor.
+// type StorageExecutorFunctions interface {
+// 	// InstanceID returns the local system's InstanceID.
+// 	InstanceID(
+// 		ctx Context,
+// 		opts Store) (*InstanceID, error)
+
+// 	// NextDevice returns the next available device.
+// 	NextDevice(
+// 		ctx Context,
+// 		opts Store) (string, error)
+
+// 	// LocalDevices returns a map of the system's local devices.
+// 	LocalDevices(
+// 		ctx Context,
+// 		opts *LocalDevicesOpts) (*LocalDevices, error)
+// }
+
+import (
+	//"bufio"
```

Comment:
Hi @akiraw,

Please remove all unused imports. If you're using Atom you could use the `go-fmt` plug-in and set it to use the `go-imports` setting which handles this action for you. Thanks!